blocks/dump.py

```diff
@@ -89,7 +89,7 @@ def extract_parameter_values(bricks):
     if isinstance(bricks, Brick):
         bricks = Selector([bricks])
     if not isinstance(bricks, Selector):
-        raise ValueError
+        bricks = Selector(bricks)
```

Comment:
I guess `bricks` can now be a list of bricks? Docstring needs to be updated then.